assets/src/edit-story/components/keyboard/index.js

```diff
@@ -117,6 +124,19 @@ export function useKeyUpEffect(
   useKeyEffect(refOrNode, keyNameOrSpec, 'keyup', callback, deps);
```

Comment:
^ Should the unused `useGlobalKeyUpEffect` be removed now? Doesn't look like it has any use anymore.

Reply:
Would it be strange to not have a useGlobalKeyUpEffect when we have a useKeyUpEffect and a useGlobalKeyDownEffect? I think it might not be consistent if someone were to use the hooks for something more custom than just listening to a keyPress.

However at the same time I'm not sure when it will get used, so I'm happy to remove this if you feel strongly. I think we should leave it in though.

Reply:
I agree. Let's leave it for now.

Reply:
I'd be in favor of removing unused code, especially since `useGlobalKeyUpEffect` is much less commonly used compared to `useGlobalKeyDownEffect`, and especially now that we have `useIsKeyPressed`, it could just be added back if it should become necessary.

However, looks like you're both in favor of keeping it though so marking as resolved, not feeling that strongly about it.